src/frontend/parser.messages

```diff
@@ -9416,7 +9416,70 @@ program: TRANSFORMEDDATABLOCK LBRACE WHILE LPAREN UPPER RPAREN UPPER VECTOR
 ## This implies that, although the LR(1) items shown above provide an
 ## accurate view of the past (what has been recognized so far), they
 ## may provide an INCOMPLETE view of the future (what was expected next).
-## In state 452, spurious reduction of production lhs -> identifier
+## In state 427, spurious reduction of production lhs -> identifier
 ##
 
 Expected identifier after type in declaration.
+
+
+program: TRANSFORMEDDATABLOCK LBRACE PROFILE WHILE
+##
+## Ends in an error in state: 299.
+##
+## nested_statement -> PROFILE . LPAREN string_literal RPAREN LBRACE list(vardecl_or_statement) RBRACE [ WHILE VOID VECTOR UPPER UNITVECTOR TRUNCATE TARGET SIMPLEX SEMICOLON ROWVECTOR RETURN REJECT REALNUMERAL REAL RBRACE PROFILE PRINT POSITIVEORDERED PLUS ORDERED OFFSET MULTIPLIER MINUS MATRIX LPAREN LOWER LBRACK LBRACE INTNUMERAL INT INCREMENTLOGPROB IF IDENTIFIER GETLP FOR ELSE COVMATRIX CORRMATRIX CONTINUE CHOLESKYFACTORCOV CHOLESKYFACTORCORR BREAK BANG ARRAY ]
+##
+## The known suffix of the stack is as follows:
+## PROFILE
+##
+
+TODO: PARSER MESSAGE NEEDED HERE.
```

Comment:
Do I just edit that message here? Never did this before so might need help.

Reply:
Yes, the message goes here. It's delimited by empty lines, which feels a bit weird to me.
The `program:` shows an example of what leads to this error state
```stan
transformed data {
  profile while
          ^ error is here
```
so something like "Expected profile name in parens."